editor/src/core/shared/template-path.ts

```diff
@@ -856,6 +856,34 @@ export function isFromSameSceneAs(a: TemplatePath, b: TemplatePath): boolean {
   return scenePathsEqual(scenePathPartOfTemplatePath(a), scenePathPartOfTemplatePath(b))
 }
 
+function dynamicScenePathToStaticScenePath(scene: ScenePath): ScenePath {
```

Comment:
Some of the below is duplicated in #971 - we decided to separate that work into its own PR and will fix the merge conflicts since this was required for this to work.